app/lib/s3_client.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+from retrying import retry
+
+
+class S3Client:
+    def __init__(self, region, bucket):
+        self.region = region
+        self.bucket = bucket
+        self.s3 = boto3.resource('s3')
+
+    @retry(wait_fixed=1000, stop_max_attempt_number=3)
```

Comment:
I'm not sure if it's a good idea to wrap multiple levels of retry logic here. Boto3 already has retries built in, what are we trying to accomplish with this layer?

If possible, we should stick to configuring the retry logic already embedded in our dependencies, instead of bringing in another one (I've never used `retrying`, but from a brief glimpse at its issue tracker, it appears unmaintained, with the last release in 2014 and a fork that is recommended to replace it).

Reply:
OK good point. I will remove this.